cli/cmd_package.go

```diff
@@ -0,0 +1,153 @@
+package cli
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/Benchkram/bob/bob"
+	"github.com/Benchkram/bob/pkg/boblog"
+	"github.com/Benchkram/bob/pkg/usererror"
+	"github.com/Benchkram/errz"
+	"github.com/spf13/cobra"
+)
+
+var packageCmd = &cobra.Command{
+	Use:   "package",
+	Short: "package manager",
+	Args:  cobra.MinimumNArgs(0),
+	Long:  ``,
+	FParseErrWhitelist: cobra.FParseErrWhitelist{
+		UnknownFlags: true,
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		var err error
+		defer errz.Recover(&err)
+
+		b, err := bob.Bob()
+		if err != nil {
+			// TODO: usererror
+			fmt.Println("can't create bob")
+			errz.Log(err)
+			return
+		}
+
+		prune, err := cmd.Flags().GetBool("prune")
+		if err != nil {
+			boblog.Log.Error(err, "parsing Flags")
+			return
+		}
+
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		// Handle prune call
+		if prune {
+			err = b.PrunePackages(ctx)
+			if err != nil {
+				if errors.As(err, &usererror.Err) {
+					boblog.Log.UserError(err)
+				} else {
+					errz.Fatal(err)
+				}
+			}
+			return
+		}
+
+		err = b.InstallPackages(ctx)
```

Comment:
Shouldn't this be done in a goroutine,  listen for ctrl+c  and cancel the context?